### PR TITLE
add magic code for room.say()  when `@bot ` happen

### PR DESCRIPTION
**Pull request by lijiarui**
#362 
add m.mentioned() for message, thanks for @JasLin tell us about magic code when @ somebody,
so add maigic code in room.say()

### CODE REVIEW

File: src/room.ts [hunk 1]
```diff
@@ -184,7 +184,8 @@ export class Room extends EventEmitter implements Sayable {
       const replyToList: Contact[] = [].concat(replyTo as any || [])
 
       if (replyToList.length > 0) {
-        const mentionList = replyToList.map(c => '@' + c.name()).join(' ')
+        const AT_SEPRATOR = String.fromCharCode(8197)
+        const mentionList = replyToList.map(c => '@' + c.name()).join(AT_SEPRATOR)
```
Comment (lijiarui) [inline src/room.ts]: no, we don't need, `AT_SEPRATOR` equals space.

> wechat use special char with unicode code 8197 to indetify real@ 
by @JasLin

**Comment by zixia** [comment]
Ok, ic. Please get at least one approvement before I could merge this PR.

Thanks!

**Comment by zixia** [comment]
Thanks all for approving!

File: src/room.ts (hunk 1 above)
Comment (zixia) [inline src/room.ts]: OK

**Pull request merged**